qutebrowser/config/configdata.py

```diff
@@ -1125,6 +1124,14 @@ def data(readonly=False):
              SettingValue(typ.QssColor(), 'black'),
              "Background color of the statusbar."),
 
+            ('statusbar.fg.private',
+             SettingValue(typ.QssColor(), '${statusbar.fg}'),
+             "Foreground color of the statusbar in private browsing mode."),
+
+            ('statusbar.bg.private',
+             SettingValue(typ.QssColor(), 'grey'),  # FIXME darker color
```

Comment:
:+1: on `#444444`! That seems pretty good to me.

I think that if some people customized their statusbar to be a certain (non-default) color in command mode for non-private windows, they would want to customize private mode as well. I think that the majority of users don't care too much (and probably wouldn't want the color to change when entering command mode), even when in private mode. I don't like the idea of having `private` trump `command`, because it might be a little confusing for some people who don't read the documentation too finely. I personally think the trumping behavior is better than the current behavior though (so the color dosen't change).